integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -415,28 +423,32 @@ private int readInternal(String path, ByteBuffer buf, long size, long offset, Fu
     final int sz = (int) size;
     int nread = 0;
     int rd = 0;
-    Long fd = fi.fh.get();
-    final int flags = fi.flags.get();
+    final long fd = fi.fh.get();
     try {
       FileInStream is = mOpenFileEntries.get(fd);
```

Comment:
Just a general question, since the fds for different opening are in different maps. There is no lock for them all?
For example, at first there would be a lookup in different maps, finally it turns out no such fd and creates one. At the same time, another one is also going thorough the same flow, and gets to create one.
Is this case handed?

Reply:
fd is assigned in each open() or create(), fd will be unique for each read or write workflow.
open for READ_WRITE open action will assign an fd in open(), and firs read or first write will be able to init the FileInStream or FileOutStream.
I added the concurrency logic in readInternal or writeInternal method to better handle the concurrency. Only one read() will create the fileInStream.